test/test_utils.go

```diff
@@ -281,3 +287,30 @@ func verifySignature(ctx context.Context, t *testing.T, c *clients, obj objects.
 		}
 	}
 }
+
+// restartChainsControllerPod restarts the pod running Chains
+// it then waits for a given timeout for the pod to resume running state
+func restartChainsControllerPod(c kubernetes.Interface, timeout time.Duration) error {
+	pods, err := c.CoreV1().Pods("tekton-chains").List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	pod := pods.Items[0]
```

Comment:
This is brittle - there's no guarantee the controller is the first pod.

You'll want to verify the name / use a label selector (e.g. `app.kubernetes.io/component=controller`)

Reply:
Good point! Done.